plugins/ngrok/credentials.go

```diff
@@ -12,14 +13,17 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+const ngrokAuthToken = "Authtoken"
+
 func Credentials() schema.CredentialType {
 	return schema.CredentialType{
 		Name:    credname.Credentials,
 		DocsURL: sdk.URL("https://ngrok.com/docs/ngrok-agent/config"),
 		Fields: []schema.CredentialField{
 			{
-				Name:                fieldname.AuthToken,
-				MarkdownDescription: "Auth Token used to authenticate to ngrok.",
+				Name:                ngrokAuthToken,
+				AlternativeNames:    []string{fieldname.AuthToken.String()},
```

Comment:
I think it would make more sense to be more explicit here:
```suggestion
				AlternativeNames:    []string{"Auth Token"},
```

Reply:
Done!